projects/m4/014-words-with-six-vowels-in-order/js/index.js

```diff
@@ -0,0 +1,48 @@
+const fs = require("fs");
+const fsPromises = fs.promises;
+const path = require("path");
+
+// TEXT FILE
+const input = path.join(__dirname, "input.txt");
+
+// MAIN FUNCTION
+
+readFile(input)
```

Comment:
I think it's better to rename the function to something other than `fs.readFile`.
Example: `getWordsSixWovels`